src/main/resources/com/google/jenkins/plugins/storage/AbstractUpload/config.jelly

```diff
@@ -43,6 +43,10 @@
   <f:entry title="${%Show inline in browser?}" field="showInline">
     <f:checkbox />
   </f:entry>
+  
+  <f:entry title="${%Metadata}" field="metadata">
+    <f:textbox default="gzip" />
```

Comment:
The new field needs a help entry.

Reply:
3 fields of this plugin doesn't have Help entry and it's not a problem, but mine is ? :)

Reply:
Yes. Adding a new field with a generic name and an unclear use case, and no help entry is problematic.
Could a user understand what this field means without digging into the code?